doc/source/user_guide/getting_help.rst

```diff
@@ -26,16 +26,6 @@ Use the ``quick search`` field in the navigation bar of the online
 documentation to find mentions of keywords (segmentation,
 rescaling, denoising, etc.) in the documentation.
 
-API Discovery
```

Comment:
Why is this taken out, if we still support it?

Reply:
Right now `lookfor` is deprecated and its removal scheduled for 0.26. While that is so, I think it makes sense to remove this section. It's also in the TODO.txt.

To be honest, I forgot that it was marked as deprecated while the discussion in https://github.com/scikit-image/scikit-image/pull/7288#discussion_r1451033431 happened. I don't mind reverting the deprecation and this in a PR if you would like to see it stay...